packages/fx-core/templates/plugins/resource/apiconnector/sample/js/aad.template

```diff
@@ -16,7 +16,7 @@ const teamsFx = new teamsfxSdk.TeamsFx(teamsfxSdk.IdentityType.App, {
 // Initializes a new axios instance to call {{config.APIName}} API.
 const appCredential = teamsFx.getCredential();
 const authProvider = new teamsfxSdk.BearerTokenAuthProvider(
-  // Please replace '<your-api-scope>' with actual api scope value.
+  // TODO: Please replace '<your-api-scope>' with actual api scope value.
```

Comment:
@timngmsft please also help review the changes here. The purpose of this prefix is explicitly tell users they need to do something here.

Reply:
TODO: Replace '<your-api-scope>' with your API scope value.